tests/api/api/test_artifacts.py

```diff
@@ -983,3 +1058,15 @@ def _generate_artifact_body(
         data["spec"] = {"body": body}
 
     return data
+
+
+def _get_artifact_url(uid: str = None, tag: str = None) -> str:
+    url = GET_API_ARTIFACT_v2_PATH.format(project=PROJECT, key=KEY)
+    params = []
+
+    if uid:
+        params.append(f"uid={uid}")
```

Comment:
IIRC It's supposed to be 'object-uid'

Reply:
We support both options, but I will change that anyway https://github.com/mlrun/mlrun/blob/7e6e4666c23f7423bd8506c1caefa0d946457ae3/server/api/api/endpoints/artifacts_v2.py#L229C8-L229C38
I will add a deprecation for this field, similar to what we did for deleting artifacts https://github.com/mlrun/mlrun/blob/7e6e4666c23f7423bd8506c1caefa0d946457ae3/server/api/api/endpoints/artifacts_v2.py#L246